src/proto/grpc/testing/control.proto

```diff
@@ -32,7 +32,7 @@ syntax = "proto3";
 import "src/proto/grpc/testing/payloads.proto";
 import "src/proto/grpc/testing/stats.proto";
 
-package grpc.testing;
+package grpc.testing.integration;
```

Comment:
If we do this, we would also need to change the directory path to match this. 

In general, I am against this change because it would mean a lot of flux for virtually no benefits.